File: 프로그래머스/lv1/12982. 예산/예산.js
function solution(d, budget) {
    d.sort((a,b)=>a-b);
    let add = 0;
    let answer = 0;
    for(let i=0; i<d.length; i++){
        add += d[i];
        if(add > budget){
            break;
        }else{
         answer++;
        }
    }
    return answer
}
              

File: 프로그래머스/1/12982. 예산/예산.java
import java.util.*;

class Solution {
    public int solution(int[] d, int budget) {
        Arrays.sort(d);
        int answer = 0;
        for(int num : d){
            if(budget-num >= 0){
                budget -= num;
                answer++;
            }
        }
        return answer;
    }
}